src/epd_driver/include/epd_driver.h

```diff
@@ -206,6 +206,30 @@ void epd_deinit();
 /** Enable display power supply. */
 void epd_poweron();
 
+#if defined(CONFIG_EPD_BOARD_REVISION_LILYGO_T5_47
```

Comment:
There's a missing `)` here, that's why the CI currently keeps failing: https://github.com/vroland/epdiy/runs/6369702620?check_suite_focus=true